documentation/release_notes.markdown

```diff
@@ -4,25 +4,66 @@ title: "Puppet Server: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
+## Puppet Server 1.1.0
+Released June 2, 2015
+
+In addition to several bug fixes, this release adds a new feature which can be configured to allow the master to automatically flush individual JRuby pool instances after a specified number of web requests have been handled. This upgrades Puppet Server's dependency on JRuby to 1.7.20 in order to take advantage of memory optimizations and other fixes.
+
+### New Features
+#### Added setting to flush ruby instances after a configurable number of requests
+We added a setting that can be used by the master to limit how many HTTP requests a given JRuby instance will handle in its lifetime. When a JRuby instance reaches this limit, it is flushed from memory and replaced with a fresh one. Defaults to 0, which disables automatic JRuby flushing. This can be useful for working around buggy module code that would otherwise cause memory leaks, however _it causes a slight performance penalty_ whenever a new JRuby has to reload all of the Puppet Ruby code. If memory leaks from module code are not an issue in your deployment, the default value will give you the best performance.
+    * [SERVER-325](https://tickets.puppetlabs.com/browse/SERVER-325)
+
+#### Allow `environment-cache` to take an environment as an argument
+Added support to the `environment-cache` API for flushing an environment by name, as opposed to only having the ability to flush all environments.
+    * [SERVER-324](https://tickets.puppetlabs.com/browse/SERVER-324) 
+
+### Bug fixes
+#### Re-enabled the master `status` endpoint
+* [SERVER-564](https://tickets.puppetlabs.com/browse/SERVER-564)
+
+#### `ignore` parameters were being mishandled
+Fix for a problem where `file_metadatas` requests to the master which include multiple `ignore` parameters were being mishandled. This had previously led to an agent downloading files from the master which should have been ignored.
+  * [SERVER-442](https://tickets.puppetlabs.com/browse/SERVER-442)
+  * [SERVER-696](https://tickets.puppetlabs.com/browse/SERVER-696)
+
+#### `keylength` value now being determined by setting
+Previously, Puppet Server had always hardcoded the `keylength` to 4096 bits. Now, it properly honors the value in the `keylength` setting when determining the number of bits to use in the generation of keys.
+* [SERVER-157](https://tickets.puppetlabs.com/browse/SERVER-157)
+
+#### Verbose output disabled
+Disabled the display of verbose output that appeared during a package upgrade.
+* [SERVER-541](https://tickets.puppetlabs.com/browse/SERVER-541)
+
+#### Previous logback level issue fix had been reverted
+Fixed an issue where logback levels weren’t changed unless you restarted Puppet Server. This functionality had been provided in the Puppet Server 1.0.2 but inadvertently removed in Puppet Server 1.0.8.
```

Comment:
I think this should be `Puppet Server 1.0.2` rather than `the Puppet Server 1.0.2`.